sdk/storage/Azure.Storage.DataMovement.Blobs/src/AppendBlobStorageResourceServiceCopyOptions.cs

```diff
@@ -19,7 +19,7 @@ public class AppendBlobStorageResourceServiceCopyOptions
         public TransferCopyMethod CopyMethod { get; set; }
 
         /// <summary>
-        /// <see cref="AppendBlobRequestConditions"/> to apply when
+        /// See <see cref="AppendBlobRequestConditions"/> to apply when
```

Comment:
Is this one not optional like the others?

Reply:
It's optional.